pkg/machinery/config/schemas/config.schema.json

```diff
@@ -1,6 +1,6 @@
 {
   "$schema": "https://json-schema.org/draft/2020-12/schema",
-  "$id": "https://talos.dev/v1.12/schemas/config.schema.json",
+  "$id": "https://raw.githubusercontent.com/siderolabs/talos/v1.12/pkg/machinery/config/schemas/config.schema.json",
```

Comment:
@utkuozdemir doesnt actually exist though 👀 I guess expected cause no `v1.12` tag yet?

https://raw.githubusercontent.com/siderolabs/talos/v1.12/pkg/machinery/config/schemas/config.schema.json

Reply:
@smira the talos.dev links dont work, that is why I made this change

e.g.

- https://www.talos.dev/v1.11/schemas/config.schema.json
- https://www.talos.dev/v1.11.5/schemas/config.schema.json

I am just fixing the links to the ones that work, matching how it is specified in omni's `.kres.yaml`

https://github.com/siderolabs/omni/blob/60cb92a12558481fa3f870dc9220c30a55c016e1/.kres.yaml#L515

Reply:
https://talossystems.slack.com/archives/C0723NUK7K6/p1764788274345549?thread_ts=1764785965.447009&cid=C0723NUK7K6